smart-contracts/scripts/devenv.ts

```diff
@@ -118,7 +118,9 @@ async function main() {
       sifnode.results.relayerAddresses[0],
       sifnode.results.witnessAddresses[0],
       golang.results,
-      hardhat.results.chainId
```

Comment:
Is there a reason this value is not able to come from the hardhat builder?

Reply:
The chainId from hardhat is 1, we need to configure it as 31337.